optuna/integration/tensorflow.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import absolute_import
+
+import optuna
+
+
+try:
+    import tensorflow as tf
+    from tensorflow.train import SessionRunHook
+    _available = True
+except ImportError as e:
+    _import_error = e
+    # PruningHook is disabled because TensorFlow is not available.
+    _available = False
+    SessionRunHook = object
+
+
+class TensorFlowPruningHook(SessionRunHook):
+    """TensorFlow SessionRunHook to prune umpromising trials.
```

Comment:
'umpromising' seems a typo for 'unpromising'.

Reply:
I fixed the typo.